src/interface/RadioButton/RadioButtonInterface.ts

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
align this section and also remove extra space

Reply:
resolved